ArcadiaParties-SPA/src/app/reducers/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  ActionReducer,
+  ActionReducerMap,
+  createFeatureSelector,
+  createSelector,
+  MetaReducer
+} from '@ngrx/store';
+import { environment } from '../../environments/environment';
+
+export interface State {
+
+}
+
+export const reducers: ActionReducerMap<State> = {
+
+};
+
+
+export const metaReducers: MetaReducer<State>[] = !environment.production ? [] : [];
```

Comment:
Do you understand what are meta reducers needed for? Probably, you won't need them in the project.
https://ngrx.io/guide/store/metareducers